### PR TITLE
initial implementation

### PR DESCRIPTION
**Pull request by trombik**

### CODE REVIEW

File: tests/serverspec/default_spec.rb [hunk 1]
```diff
@@ -3,47 +3,78 @@
...
-config  = "/etc/sensu-api/sensu-api.conf"
-user    = "sensu-api"
-group   = "sensu-api"
-ports   = [PORTS]
-log_dir = "/var/log/sensu-api"
-db_dir  = "/var/lib/sensu-api"
+config_dir = "/etc/sensu"
+user    = "sensu"
+group   = "sensu"
+ports = [4242]
```
Comment (mitsururike) [inline tests/serverspec/default_spec.rb:9]: it seems it does not have to be an array unless other ports are planned to be added.

File: tests/serverspec/default_spec.rb [hunk 2]
```diff
@@ -3,47 +3,78 @@
...
-    it { should be_file }
-  end
+describe file("#{config_dir}/conf.d/api.json") do
+  it { should exist }
+  it { should be_file }
+  it { should be_mode 644 }
+  it { should be_owned_by default_user }
+  it { should be_grouped_into default_group }
+  its(:content_as_json) { should include("api" => include("host" => "10.0.2.15")) }
+  its(:content_as_json) { should include("api" => include("bind" => "10.0.2.15")) }
```
Comment (mitsururike) [inline tests/serverspec/default_spec.rb]: define `ipaddress = "10.0.2.15" ` and use `#{ipaddress}` is better if it is to be used in other locations.

File: tests/serverspec/default_spec.rb (hunk 1 above)
Comment (trombik) [reply tests/serverspec/default_spec.rb:9]: correct. but i have not investigated every possible configuration and am not sure other plugins, or whatever, have capabilities to listen.

besides, it does not harm anything.

File: tests/serverspec/default_spec.rb [hunk 3]
```diff
@@ -3,47 +3,78 @@
...
+  its(:content_as_json) { should include("api" => include("port" => 4242)) }
```
Comment (mitsururike) [inline tests/serverspec/default_spec.rb]: `port = 4242` and use `#{port}` is better if it is the same port to be listened.

**Review by mitsururike — APPROVED** [review]

**Pull request merged**